### ISSUE TITLE
`[error] require() of ES Module`

### ISSUE THREAD

**Issue by thought2** [ts-graphviz/prettier-plugin-dot#21]
Unfortunately I cannot run the plugin, I'm getting a `[error] require() of ES Module` error.

Here's the complete CLI session:

```
$ node --version
v19.1.0

$ yarn --version
1.22.19

$ yarn init -y
[..]
Done in 0.04s.

$ yarn add -D prettier prettier-plugin-dot
[...]
Done in 1.95s.

$ yarn run prettier --write '**/*.dot'
yarn run v1.22.19
$ {{PROJECT_DIR}}/node_modules/.bin/prettier --write '**/*.dot'
[error] require() of ES Module {{PROJECT_DIR}}/node_modules/prettier-plugin-dot/lib/index.js from {{PROJECT_DIR}}/node_modules/prettier/index.js not supported.
[error] Instead change the require of {{PROJECT_DIR}}/node_modules/prettier-plugin-dot/lib/index.js in {{PROJECT_DIR}}/node_modules/prettier/index.js to a dynamic import() which is available in all CommonJS modules.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by kamiazya** [comment]
@all-contributors please add @thought2  for bug

**Comment by kamiazya** [comment]
@thought2 
Sorry it took me so long to notice.

The package.json was incorrectly configured.

The corrected version has been released as version [0.4.1](https://github.com/ts-graphviz/prettier-plugin-dot/releases/tag/v0.4.1).
